File: system-test/dotnet/SystemTests/Legacy/Mod09/SmokeTests/External/TaxSmokeTest.cs
using SystemTests.Legacy.Mod09.Base;
using Xunit;

namespace SystemTests.Legacy.Mod09.SmokeTests.External;

public class TaxSmokeTest : BaseScenarioDslTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToTax()
    {
        await Scenario().Assume().Tax().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod09/smoke/external/TaxSmokeTest.java
package com.mycompany.myshop.systemtest.legacy.mod09.smoke.external;

import com.mycompany.myshop.systemtest.legacy.mod09.base.BaseScenarioDslTest;
import org.junit.jupiter.api.Test;

class TaxSmokeTest extends BaseScenarioDslTest {
    @Test
    void shouldBeAbleToGoToTax() {
        scenario.assume().tax().shouldBeRunning();
    }
}
